projects/keycloak/exported-realm.json

```diff
@@ -0,0 +1,3175 @@
+[ {
```

Comment:
Was there any changes to the [secure-app realm](https://github.com/feedhenry/mobile-security/blob/master/projects/keycloak/secure-app-realm.json) or is this new realm file being used too? 

I see this config is also bringing in the master realm which we will cause the import to fail with a duplicate realm.

Reply:
My mistake. This file wasn't meant to checked in. Will remove it.